cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -4,16 +4,23 @@
 from pathlib import Path
 
 from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import SequencingFileTag
 from cg.constants.devices import DeviceType
-from cg.exc import MissingFilesError, FlowCellError
+from cg.exc import FlowCellError, MissingFilesError
+from cg.meta.demultiplex.housekeeper_storage_functions import (
+    add_demux_logs_to_housekeeper,
+    add_run_parameters_file_to_housekeeper,
+    add_sample_fastq_files_to_housekeeper,
+    store_undetermined_fastq_files,
```

Comment:
I think at least some of these need to change as well. I would copy over the ones we need to a `storage_functions` file in here

Reply:
I will resolve this as many changes were made, set for another review